refinery/core/views.py

```diff
@@ -549,51 +519,9 @@ def solr_core_search(request):
             if annotations:
                 response['response']['annotations'] = annotation_data
 
-    return JsonResponse(response)
-
+            return JsonResponse(response)
```

Comment:
Is this intentional? Unless `allIds or annotations` and `response_ids.status_code == 200` it should return `None`

Reply:
oh, sorry, just scrolled down and saw the `HTTPResponse`